tracker/tracker/src/main/app/observer/observer.ts

```diff
@@ -270,6 +270,11 @@ export default abstract class Observer {
             ),
           );
         }
+        if (this.app.sanitizer.isMaskedContainer(id)) {
+          this.sendNodeAttribute(id, node, "style",
```

Comment:
This will be overwritten if the original node has a `style` attribute. 
The best way would be having a special message type for that.
You can go here with a shallow cloning so far since this masking is a pretty rare case.

Reply:
Also no need for `background` - the style for such elements can be set by data-openreplay-htmlmasked attribute on the player end.